src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -654,6 +649,26 @@ export default {
 	justify-content: flex-start !important;
 }
 
+.button-container {
```

Comment:
Also, styles shouldn't be neccessary, when you insert `NcButton` component

Reply:
that's the container of the two buttons. What do you think? should it be a button filter or the 3 dots on the right that drops a list of the two filters with the conversation addition option?

Reply:
Sorry, I meant not the container class, but all new styles for buttons (as NcButton has it by default)

I'm thinking, that we should take as little space as possible for this feature, so three-dots menu seems more preferable to me. But we could also wait for design team proposal, or suggest our option in specific issue (you could put your screenshots there)